package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@luckbox/ipc-messenger",
   "author": "Luckbox",
-  "version": "2.0.1",
```

Comment:
This version change needs to be reflected in the `package-lock.json` file as well.